app/policies/business_category_policy.rb

```diff
@@ -4,30 +4,24 @@ def new?
     user.admin?
   end
 
-
```

Comment:
@weedySeaDragon Just thought I'd delete them since I am used to having just one space just like with other projects. 

I have put them back now to having two spaces. Only modified instance variables in this PR and changed them to readers. Thanks.